### PR TITLE
modify create-bundle to work on MacOS

### PR DESCRIPTION
**Pull request by RomanBednar**
Changes create-bundle script similar to what we use in LSO.

cc @openshift/storage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ EOF
 make
 
 # Set the environment variables
+export OPERATOR_NAME=aws-efs-csi-driver-operator
```
Comment (dobsonj) [inline README.md]: I already see `OPERATOR_NAME` on line 32

**Comment by dobsonj** [comment]
One minor nit on the README, but the create-bundle changes look good to me.

File: README.md (hunk 1 above)
Comment (RomanBednar) [reply README.md]: I managed to miss that somehow, dropping this change.

**Comment by dobsonj** [comment]
/lgtm
/approve

This PR has no impact on customers, docs, qe, ci... just makes our lives easier when testing development changes.
/label docs-approved
/label px-approved
/label qe-approved
/label acknowledge-critical-fixes-only

**Pull request merged**